ci/nightly/pipeline.template.yml

```diff
@@ -533,16 +533,17 @@ steps:
               composition: persist
               args: [--node-count=4, --consensus=cockroach, --blob=maelstrom, --time-limit=300, --concurrency=4, --rate=500, --max-txn-length=16, --unreliability=0.1]
 
-      - id: persist-txn-maelstrom
-        label: Maelstrom coverage of persist-txn
-        timeout_in_minutes: 10
-        agents:
-          queue: linux-x86_64
-        artifact_paths: [test/persist/maelstrom/**/*.log, junit_*.xml]
-        plugins:
-          - ./ci/plugins/mzcompose:
-              composition: persist
-              args: [--consensus=mem, --blob=mem, --persist-txn]
+      # TODO(txn): Get this turned back on.
```

Comment:
```suggestion
      # TODO(txn): Get this turned back on before enabling the feature by default/in production.
```

Reply:
That's actually exactly what the `TODO(txn)` marker means! :D